bases/ecoindex/backend/routers/ecoindex.py

```diff
@@ -56,10 +47,10 @@
 )
 async def get_ecoindex_analysis_list(
     response: Response,
-    host: Annotated[str, Depends(host_parameter)],
-    version: Annotated[Version, Depends(version_parameter)] = Version.v1,
-    date_range: Annotated[DateRange, Depends(date_parameters)] = DateRange(),
-    pagination: Annotated[Pagination, Depends(pagination_parameters)] = Pagination(),
+    host: HostParameter,
```

Comment:
Consider using a more descriptive name for the `host` parameter in the `get_ecoindex_analysis_list` function. The current name does not clearly indicate what the parameter represents. A more descriptive name, such as `host_name`, would improve code readability. [medium]

Reply:
For legacy issues, I have to let it as it...